skiko/src/jvmMain/kotlin/org/jetbrains/skiko/HardwareLayer.kt

```diff
@@ -15,20 +16,35 @@ open class HardwareLayer : Canvas(), Drawable {
     override fun paint(g: Graphics) {
         display()
     }
+
+    private val platformOperations by lazy { getPlatformOperations() }
```

Comment:
Make that a top-level property, and get rid of property here. It is always the same

Reply:
Done